src/pages/Booking/MusicalBooking/TimeItem.js

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { useBookingStore } from '../store/store';
+import { shallow } from 'zustand/shallow';
+
+function TimeItem({
+  id,
+  name,
+  time,
+  endTime,
+  totalSeats,
+  availableSeats,
+  handleDynamicFetch,
+}) {
+  const [selectedMusical, selectedTime] = useBookingStore(
+    state => [
+      state.bookingState.selectedMusical,
+      state.bookingState.selectedTime,
+    ],
+    shallow
+  );
+  const updateTime = useBookingStore(state => state.setTime);
+
+  const handleTimeClick = timeInfo => {
+    updateTime(timeInfo);
+  };
+  return (
+    <TimeItemContainer
+      isSelected={id === selectedTime?.id ? true : false}
+      onClick={() => {
+        handleDynamicFetch('timeId', id);
+        handleTimeClick({ id, name, time });
+      }}
```

Comment:
```suggestion
      onClick={() => {
        handleDynamicFetch('timeId', id);
        handleTimeClick({ id, name, time });
      }}
```
TheatreItem.js 리뷰와 동일합니다

Reply:
반영 완료했습니다! 리뷰 감사합니다 🙌